src/buoyancy.jl

```diff
@@ -6,15 +6,29 @@ const g_Earth = 9.80665
 Supported buoyancy types:
 
 - Nothing 
+- BuoyancyTracer
 - SeawaterBuoyancy
 =#
 
+validate_buoyancy(::Nothing, tracers) = nothing
+
+function validate_buoyancy(buoyancy, tracers) 
+    req_tracers = required_tracers(buoyancy)
+
+    all(tracer ∈ tracers for tracer in req_tracers) || 
+        error("$(req_tracers) must be among the list of tracers to use $(typeof(buoyancy).name.wrapper)")
+
+    return nothing
+end
```

Comment:
Seems like it would make sense to return `true` if `buoyancy` and `tracers` match?

Reply:
You mean if `buoyancy` is a tuple of symbols?